### ISSUE TITLE
Checkbox Focus in Login Block

### ISSUE THREAD

**Issue by adane** [Tizra/Tizra-Customer-Tracker#258]
From Henry:

We have a problem with checkbox focus that may need Dave B's review.

In the Login smartblock, the native OS checkbox is replaced with a custom graphical checkbox. The OS checkbox is hidden and the graphical checkbox is managed with CSS.

The problem is that the graphical checkbox doesn't take focus.

This could be solved by:

a) Assigning the graphical checkbox to an anchor tag rather than a span (anchor tags always take focus) or a tabindex property.
b) Reverting to the standard OS checkbox (my preference).

For case (a) we need to ensure that the once the object takes focus, the javascript can set and unset it using keyboard events (space / tab).

Thanks,

Henry

**Comment by adane** [comment]
Question on this: Would Henry's preferred approach of reverting to the OS checkbox result in something that wouldn't fit with the rest of the design visually?

**Comment by dbarberich** [comment]
I don't think it will be too noticeable. With the move to Foundation 4, there was a conflict with getting the selects to be styled the Foundation way, so those are currently OS versions rather than Foundation versions. So if we move ahead with this checkboxes will be too. Other form fields can stay as they are.
